jemoji.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'jekyll', '>= 3.0'
   s.add_dependency 'html-pipeline', '~> 2.2'
   s.add_dependency 'activesupport', '~> 4.0'
-  s.add_dependency 'gemoji', '~> 2.0'
+  s.add_dependency 'gemoji', '~> 3.0'
 
```

Comment:
Since GitHub is still hosting the images, I think we're in the clear here. There may be certain emojis that GitHub isn't hosting, but IMHO that's kind of out of our control and Gemoji provides facilities for extracting images.